linode/rdns/resource_test.go

```diff
@@ -206,6 +206,32 @@ func TestAccResourceRDNS_waitForAvailableWithTimeout(t *testing.T) {
 	})
 }
 
+func TestAccResourceRDNS_unreservedToReserved(t *testing.T) {
+	t.Parallel()
+
+	resName := "linode_rdns.foobar"
+	linodeLabel := acctest.RandomWithPrefix("tf_test")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acceptance.PreCheck(t) },
+		ProtoV5ProviderFactories: acceptance.ProtoV5ProviderFactories,
+
+		Steps: []resource.TestStep{
+			{
+				Config: tmpl.UnreservedToReserved(t, linodeLabel, testRegion, true),
```

Comment:
This fixture seems to be missing 👀

Reply:
This fixture doesnt belong in this resource. It has now been moved to linode_networking_ip resource.